tasks/kernel_matrix_testing/init_kmt.py

```diff
@@ -123,3 +124,8 @@ def init_kernel_matrix_testing_system(ctx, lite):
         download_rootfs(ctx, KMT_ROOTFS_DIR, KMT_BACKUP_DIR)
         download_kernel_packages(ctx, KMT_PACKAGES_DIR, KMT_KHEADERS_DIR, KMT_BACKUP_DIR)
         gen_ssh_key(ctx)
+
+    # build docker compile image
+    ctx.run("cat /proc/$$/status | grep '^Groups:' | grep $(cat /etc/group | grep 'docker:' | cut -d ':' -f 3)")
```

Comment:
```suggestion
    ctx.run("groups | grep 'docker'")
```

Reply:
`groups` uses `/etc/group` to return the membership of the user. However, this can cause problems in the cases where you add a user to a new group, thus updating `/etc/group` but the changes have not taken effect for the current session. As a results `groups` would succeed but subsequent commands would fail because the current session of the user does not have the appropriate membership.